postgres/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 * [IMPROVEMENT] Deprecating "postgres.replication_delay_bytes" in favor of "postgresql.replication_delay_bytes". See[#639][] and [#699][], thanks to [@Erouan50][]
 * [MINOR] Allow specifying postgres port as string. See [#607][], thanks [@infothrill][]
+* [IMPROVEMENT] Add option for including the default database 'postgres' when gathering stats [#740][]
```

Comment:
Create a new section in the changelog with a new version and `Unreleased` for the date.
And I would rewrite this line:
"Add an option to include the default 'postgres' database when gathering stats"

Reply:
Oh, this must have happened when updating the branch, thanks!